linux_os/guide/system/logging/ensure_rsyslog_log_file_configuration/rsyslog_encrypt_offload_actionsendstreamdriverauthmode/rule.yml

```diff
@@ -0,0 +1,47 @@
+documentation_complete: true
+
+title: Ensure Rsyslog Authenticates Off-Loaded Audit Records
+
+description: |-
+    Rsyslogd is a system utility providing support for message logging. Support
+    for both internet and UNIX domain sockets enables this utility to support both local
+    and remote logging.  Couple this utility with <tt>gnutls</tt> (which is a secure communications
+    library implementing the SSL, TLS and DTLS protocols), and you have a method to securely
+    encrypt and off-load auditing.
+
+    When using <tt>rsyslogd</tt> to off-load logs the remote system must be authenticated.
+
+rationale: |-
+    The audit records generated by Rsyslog contain valuable information regarding system
+    configuration, user authentication, and other such information. Audit records should be
+    protected from unauthorized access.
+
+severity: medium
+
+identifiers:
+    cce@rhel8: CCE-86339-9
+
+references:
+    disa: CCI-001851
+    nist: AU-4(1)
+    srg: SRG-OS-000342-GPOS-00133,SRG-OS-000479-GPOS-00224
+    stigid@rhel8: RHEL-08-030720
+
+
+ocil_clause: '$ActionSendStreamDriverAuthMode in /etc/rsyslog.conf is not set to x509/name'
+
+ocil: |-
+    Verify the operating system authenticates the remote logging server for off-loading audit logs with the following command:
+
+    <pre>$ sudo grep -i '$ActionSendStreamDriverAuthMode' /etc/rsyslog.conf /etc/rsyslog.d/*.conf</pre>
+    The output should be
+    <pre>$/etc/rsyslog.conf:$ActionSendStreamDriverAuthMode x509/name</pre>
+
+
+template:
+    name: lineinfile
```

Comment:
I probably don't understand why there's a template section here if you are providing `shared` content

Reply:
That looks like a leftover from a previous attempt, removed.